RPIPs/RPIP-46.md

```diff
@@ -28,42 +28,47 @@ A fixed percentage of RPL inflation is currently being used to fund ongoing main
    2. Legacy minipools are an exception and SHALL continue to support earlier distribution methodologies 
-6. `node_operator_commission_share`, `surplus_share`, `increase_no_share_seal_increment`, `increase_no_share_seal_count`, and `allowlisted_controllers` SHALL be updateable by pDAO vote
-7. `node_operator_commission_share` and `surplus_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
+5. `node_operator_commission_share` and `surplus_share`, SHALL be updateable by an address in the `allowlisted_controllers` array
    1. This functionality SHALL not be used without a separate pDAO vote to enable a controller and add it to the list
-8. The security council SHALL have a limited-use power to increase the `node_operator_commission_share` by `increase_no_share_seal_increment` and decrease the `surplus_share` by the same amount
-   1. This power SHALL be usable if `increase_no_share_seal_count` > 0
-   2. `increase_no_share_seal_count` SHALL be decremented by one upon using this power
-   3. This power SHOULD be used if the deposit pool is over half-full for the majority of a 2-week period at the current `node_operator_commission_share`
-   4. The pDAO SHALL be able to change `increase_no_share_seal_count` via vote
-9. The initial settings SHALL be:
-   1. `node_operator_commission_share`: 3.5%
-   2. `voter_share`: 5%
-   3. `surplus_share`: 5.5%
-   4. `increase_no_share_seal_increment`: 0.5%
-   5. `increase_no_share_seal_count`: 6
-   6. `allowlisted_controllers`: []
-10. Prior to the release of Saturn 1, a ranked-choice vote MUST be held to select a mechanism for surplus revenue share
-    1. The choices MAY include [RPIP-45: RPL Burn](RPIP-45.md), [RPIP-50: RPL LP](RPIP-50.md), or using higher `voter_share`
-    2. If higher `voter_share` is selected, the entirety of the ["Specification taking effect with Saturn 2"](#specification-taking-effect-with-saturn-2) section below SHALL be deleted
-    3. The selected mechanism MAY be implemented in Saturn 1
-       1. If not, the revenue SHALL be held in reserve until the mechanism is implemented. In such a case, once the mechanism is implemented, the revenue SHALL be distributed in the same amount of time it took to build up, or faster.
-    4. The selected mechanism MUST be implemented in Saturn 2
-    5. When implemented, `surplus_share` SHOULD be renamed to something more specific
+6. The `node_operator_commission_share_council_adder` setting SHALL only be set to values within 0% and `max_node_operator_commission_share_council_adder` (inclusive)
+7. The `node_operator_commission_share_council_adder` setting SHALL be controllable by the security council without requiring a delay
+8. The security council SHOULD increment `node_operator_commission_share_council_adder` by 0.5% if the deposit pool is over half-full for the majority of a 2-week period with a constant `node_operator_commission_share + node_operator_commission_share_council_adder`
```

Comment:
Generally if we have the additionally complexity of council controlled settings, I think we might as well allow full freedom between the range (take advantage of letting it also be used to lower the NO_share if the NO queue is consistently too long). But in either case I think we either use "MAY"/move to optional heuristic section for flexibility. Or if we want it to work strictly in a similar way as the seals did, we should update this to "SHALL". Leaving it here as "SHOULD" could be confusing for the security council to decide if a specific situation is worth doing something different than "SHOULD"

Reply:
Or maybe delineate “SHALL” for increment by 0.5%, but “SHOULD” for 2 week deposit pool thing

Reply:
Going with:
```
8. The security council SHOULD increment `node_operator_commission_share_council_adder` by 0.5% if the deposit pool is over half-full for the majority of a 2-week period with a constant `node_operator_commission_share + node_operator_commission_share_council_adder`
    1. The security council SHALL NOT otherwise change `node_operator_commission_share_council_adder`
```

Reply:
Makes sense...